xarray/convert.py

```diff
@@ -3,24 +3,36 @@
 import numpy as np
 
 from .core.dataarray import DataArray
+from .core.pycompat import OrderedDict
 from .conventions import (
     maybe_encode_timedelta, maybe_encode_datetime, decode_cf)
 
-ignored_attrs = set(['name', 'tileIndex'])
+cdms2_ignored_attrs = {'name', 'tileIndex'}
```

Comment:
Currently we support Python 2.6, so set literals will cause test failures.

I think we can change this for the next major release of xarray though -- I just sent out a mailing list post about this.